Cargo.toml

```diff
@@ -21,7 +21,7 @@ exclude = [".*"]
 [dependencies]
 libz-sys = { version = "1.1.20", optional = true, default-features = false }
 libz-ng-sys = { version = "1.1.16", optional = true }
-libz-rs-sys = { version = "0.2.1", optional = true, default-features = false, features = ["std", "rust-allocator"] }
```

Comment:
Is there a good reason for removing the explicit handling of features? It looks like [they didn't change](https://docs.rs/crate/libz-rs-sys/latest/features).

The rationale for having them is to not rely on default feature remaining stable but instead take note consciously on which features are enabled.
I'd prefer to keep that.